sdk/helper/tokenhelper/tokenhelper.go

```diff
@@ -101,6 +104,11 @@ func TokenFields() map[string]*framework.FieldSchema {
 			Type:        framework.TypeDurationSecond,
 			Description: "The initial ttl of the token to generate",
 		},
+
+		"token_max_num_uses": &framework.FieldSchema{
+			Type:        framework.TypeInt,
+			Description: "The maximum number of times a token issued from this role may be used.",
```

Comment:
I think we should add to the description that in order to remove the limit/max, to set this value to `0`, assuming that's valid